{"classFile":"MathExceptionTest","className":"MathExceptionTest","packages":"org.apache.commons.math","methodInfos":[{"md5":"xtFr4DvdeZmwZkKKAMl+eA==","methodName":"testConstructor","parameters":"[]"},{"md5":"/AxZupYKzVq2T0IplCOXRg==","methodName":"testConstructorCause","parameters":"[]"},{"md5":"UJ7KiaUeA6geMGork0n3Mg==","methodName":"testPrintStackTrace","parameters":"[]"},{"md5":"nmHPDPmL2YzkXD4sHgZ2tg==","methodName":"testSerialization","parameters":"[]"}],"addLines":[28,29,34,36,39,40,41,44,49,55,68,69,74,75,81,110,112,113],"delLines":[20,29,34,38,39,40,47,65,66,71,72,106,108,109],"type":"MODIFY","path":"src/test/java/org/apache/commons/math/MathExceptionTest.java","pro_Commit":"e41d0d67e3b4d049657fe9e3591cfc782820e52e","pro_Date":"2011-03-16 19:07:05","sha1":"47e1e4412ba5c71c116366a53d0ab066d67d9413","file_Date":"2011-03-21 01:24:14","projectName":"commons-math","commitID":"96c2ce398318fdc1c4ebb865ef06061e72c6e5fd"}